UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/FunctionalTests/POSEndToEndValidFunctionalScenarios.cs

```diff
@@ -6,7 +6,8 @@
 
 namespace UKHO.PeriodicOutputService.API.FunctionalTests.FunctionalTests
 {
-    public class POSEndToEndFunctionalScenarios
+    [Category("POSEndToEndValidFunctionalScenarios")]
+    public class POSEndToEndValidFunctionalScenarios
```

Comment:
As discussed with Jason, we have a reason to keep all scenarions as separate files. For each test case scenario, we run the webjob and everytime stub creates new batches according to given scenario.

Category attribute is added to run all test case files sequentially instead of parallel. Otherwise, we get 409-conflict from webjob.